### PR TITLE
add mesa-tkg-git, simplify kernel & wine variation, small fixes

### PR DESCRIPTION
**Pull request by BangL**

### CODE REVIEW

File: home/main-builder/build-tkg [hunk 1]
```diff
@@ -180,6 +217,15 @@ if [ -z "$TKG_NOVK" ]; then
 	cd ..
 fi
 
+# - mesa
+if [ -z "TKG_NOMESA" ]; then
```
Comment (PedroHLC) [inline home/main-builder/build-tkg]: missing ```$```

File: home/main-builder/build-tkg [hunk 2]
```diff
@@ -180,6 +217,15 @@ if [ -z "$TKG_NOVK" ]; then
...
+	cd mesa-git || exit
+	~/chaotic-makepkg --noconfirm -C 2>&1 | tee "$TKGLOGS/mesa-git.log"
+	~/chaotic-add
+	unset TARGET_EXTRAPKGS
```
Comment (PedroHLC) [inline home/main-builder/build-tkg]: Not needed, TARGET_EXTRAPGS was not set

File: home/main-builder/build-tkg [hunk 3]
```diff
@@ -123,7 +162,7 @@ if [ -z "$TKG_NOWINE" ]; then
 		s/_EXTERNAL_INSTALL_TYPE=\"[^\"]*\"/_EXTERNAL_INSTALL_TYPE=\"opt\"/g
 		" customization.cfg
 
-		echo "[build-tkg] customization setted to: ${_PKGNAME/%-git/-opt-git}"
+		echo "[build-tkg] customization set to: ${_PKGNAME/%-git/-opt-git}"
```
Comment (PedroHLC) [inline home/main-builder/build-tkg:165]: Thanks for the typo fix :smile:

**Review by PedroHLC — CHANGES_REQUESTED** [review]
Once again: Thanks for everything :sweat_smile:

**Comment by BangL** [comment]
fixed.

**Review by PedroHLC — APPROVED** [review]

**Pull request merged**